content/en/getting-started/auth-token/index.md

```diff
@@ -67,7 +77,7 @@ $ docker run \
 
 For more information about starting LocalStack with Docker, take a look at our [Docker installation](https://docs.localstack.cloud/getting-started/installation/#docker) guide.
 
-### Starting LocalStack via Docker-Compose
+### Docker Compose
 
 To start LocalStack using `docker-compose`, you have to include the `LOCALSTACK_AUTH_TOKEN` environment variable in your `docker-compose.yml` file:
```

Comment:
```suggestion
To start LocalStack using `docker compose`, you have to include the `LOCALSTACK_AUTH_TOKEN` environment variable in your `docker-compose.yml` file:
```

We should use the new version of docker compose. https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command